src/xenia/kernel/kernel_state.cc

```diff
@@ -37,6 +37,8 @@
 #include "third_party/crypto/TinySHA1.hpp"
 
 DEFINE_bool(apply_title_update, true, "Apply title updates.", "Kernel");
+DEFINE_bool(ignore_incompatible_title_update, false,
```

Comment:
Maybe change it to ``allow_incompatible_title_update``?
It should be less confusing imo